aten/src/ATen/native/native_functions.yaml

```diff
@@ -6652,7 +6652,7 @@
   device_check: NoCheck   # TensorIterator
   variants: method
 
-- func: cross_entropy_loss(Tensor self, Tensor target, Tensor? weight=None, int reduction=Mean, int ignore_index=-100) -> Tensor
+- func: cross_entropy_loss(Tensor self, Tensor target, Tensor? weight=None, int reduction=Mean, int ignore_index=-100, float label_smoothing=0.0) -> Tensor
```

Comment:
FYI I think it's likely we'll need to keep the old signature around as well for FC reasons.